### ISSUE TITLE
renderer disconnected

### ISSUE THREAD

**Issue by eiyike123** [javieryu/nerf_bridge#30]
After running the ros_train. it trains but i can visualize the rendering. 
![Screenshot from 2023-04-13 20-46-03](https://github.com/javieryu/nerf_bridge/assets/128630463/d1c68d3a-16bd-4c44-a0e4-ac47a240ffd3)

**Comment by javieryu** [comment]
I believe you brought this up in another issue. This is likely related to networking with Docker. Does nerfstudio (no nerf_bridge) work in your docker container?

**Comment by eiyike123** [comment]
Yes nerfstudio works in my docker container. It’s training but I can view
